src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,13 @@
+$cell-size: 100px;
+$base-color: #eee;
+$month-length: 28;
```

Comment:
this variable is not required as dont have to use it anywhere

Reply:
done.